packages/scoped-elements/src/transform.js

```diff
@@ -21,9 +22,9 @@ const re = new RegExp(`<\\/?([a-z](${chars})*-(${chars})*)`, 'g');
 /**
  * The global cache of processed string arrays
  *
- * @type {Map<TemplateStringsArray, TemplateStringsArray>}
+ * @type {Cache<TemplateStringsArray, TemplateStringsArray>}
  */
-const globalCache = new Map();
+const globalCache = new Cache();
```

Comment:
since this is a global, should the cache use a weakmap?

Reply:
Agree, I'm doing the change

Reply:
oh, WeakMap can't be used as it doesn't allow strings as keys and requires objects.